templates/splash/splashpage-contextual-en.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: splashpage-en
+title: Canada.ca
+lang: en
+description-en: The Government of Canada website is a single point of access to all programs, services, departments, ministries and organizations of the Government of Canada.
+description-fr: Le site Web du gouvernement du Canada fournit un point d'accès complet à tous les programmes, services, départements, ministères et organismes du gouvernement du Canada.
```

Comment:
Not urgent, but this generic description would require to be adapted to the context